its/src/test/java/com/sonar/it/scanner/msbuild/VBNetTest.java

```diff
@@ -1,152 +0,0 @@
-/*
```

Comment:
The reason of deleting this test class is that after implementing common orchestrator, c# and vb.net plugins should be added together in the init of the orchestrator, thus failing the "vbnet only" test on the ncloc assertion. This test was also implicitely already done with a test in msbuildtest class, and the last one was moved in this class as well.